WooCommerce/WooCommerceTests/ViewRelated/Products/ProductDetailsFactoryTests.swift

```diff
@@ -46,35 +24,14 @@ final class ProductDetailsFactoryTests: XCTestCase {
 
     // MARK: External/affiliate product type
 
-    func test_factory_creates_product_form_for_affiliate_product_when_products_release_3_is_on() {
+    func test_factory_creates_product_form_for_affiliate_product() {
```

Comment:
The `expectation` below (`Wait for loading Products feature switch from app settings`) should be updated as well (2 of 2)